### PR TITLE
Tt 344 add pagination where feature is not available

### PR DESCRIPTION
**Pull request by DrFreud1**
### Description

Pagination option was added to **Time entries** and **Reports** tabs, where it was not previously available. Also, test for new methods were created.

### CODE REVIEW

**Comment by EdansRocks** [comment]
Hi Santiago! I was testing your PR and I realized that in the Time Entries section the table brings the content upside down, ie, the first row is the first day of this month and the last is the current day and this should be reversed, maybe you forgot some attribute?

image

**Comment by EdansRocks** [comment]
I also noticed that now when editing an entry in the Time Entries section I get this error and the page gets stuck.

image

**Comment by JosueOb** [comment]
Hi Santiago, I report that in the time entries section when selecting the calendar option and then returning to the list of entries, the pagination is not displayed.
Captura de Pantalla 2021-09-21 a la(s) 16 47 08
Captura de Pantalla 2021-09-21 a la(s) 16 47 41

File: src/app/modules/time-entries/pages/time-entries.component.spec.ts [hunk 1]
```diff
@@ -133,6 +133,13 @@ describe('TimeEntriesComponent', () => {
     });
   }));
 
+  it('after the component is initialized it should initialize the table', () => {
```
Comment (josepato87) [inline src/app/modules/time-entries/pages/time-entries.component.spec.ts:136]: Maybe the description of this test could be "should initialize the table when the component is initialized'"

**Review by josepato87 — CHANGES_REQUESTED** [review]

File: src/app/modules/time-entries/pages/time-entries.component.spec.ts (hunk 1 above)
Comment (DrFreud1) [reply src/app/modules/time-entries/pages/time-entries.component.spec.ts:136]: I agree, your proposal is more descriptive. I have already made the change.

**Review by josepato87 — APPROVED** [review]
LGTM

**Comment by JosueOb** [comment]
Hi Santiago, I report that in the time entry section when you select a month that has no entries and then go back to a month that has entries, the pagination is not displayed.
Captura de Pantalla 2021-09-28 a la(s) 19 25 25
Captura de Pantalla 2021-09-28 a la(s) 19 26 29

Captura de Pantalla 2021-09-28 a la(s) 19 27 10

**Pull request merged**